### PR TITLE
webView: Fix message list resizes when autocomplete pops.

### PR DESCRIPTION
**Pull request by jainkuniya**
This PR
screen shot 2018-01-17 at 6 06 35 pm

on master 4e370802a1d0cc6dfe78a8415694a2e44e5a72f7
screen shot 2018-01-17 at 6 07 31 pm

### CODE REVIEW

File: src/app/appActions.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /* @flow */
-import type { Action, Dimensions, Dispatch, GetState } from '../types';
```
Comment (borisyankov) [inline src/app/appActions.js]: Why did you rename this type?

Comment (jainkuniya) [reply src/app/appActions.js]: I want to import `Dimensions` from `react-native` (https://github.com/jainkuniya/zulip-mobile/blob/50aa4611fb75e6aece3801137c94b5e85a5fad03/src/compose/ComposeBox.js#L3) but `Dimensions` was already imported from `types` (https://github.com/zulip/zulip-mobile/blob/master/src/compose/ComposeBox.js#L13)

Comment (borisyankov) [reply src/app/appActions.js]: Then, rename this to `DimensionsType`?
`SafeAreaDimensions` is just not correct.

Comment (borisyankov) [reply src/app/appActions.js]: That being said, I don't think the `Dimensions.get()` is even the correct approach.
Just setting an `absolute` position for the popup and aligning it to the bottom is likely going to work better.

**Comment by borisyankov** [comment]
Fixed in #1757
Always consider that a bug fix might be simple and should likely not require lots of changes to the code.
That way, it is less effort to fix, less effort to review, and more likely to not introduce new issues.
